internal/venice-test-common/src/integrationTest/java/com/linkedin/venice/integration/utils/VeniceServerWrapper.java

```diff
@@ -227,6 +232,11 @@ static StatefulServiceProvider<VeniceServerWrapper> generateService(
         serverPropsBuilder.put(KafkaTestUtils.getLocalCommonKafkaSSLConfig(SslUtils.getTlsConfiguration()));
       }
 
+      if (isGrpcEnabled) {
```

Comment:
What if in the future we want to turn on gRPC by default and in the tests we want to turn it off?

IMO this block should be rewritten as
```java
      serverPropsBuilder.put(ENABLE_GRPC_READ_SERVER, isGrpcEnabled);
      if (isGrpcEnabled) {
        serverPropsBuilder.put(GRPC_READ_SERVER_PORT, TestUtils.getFreePort());
      }
```